gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/exception/PubSubMessageReceiverException.java

```diff
@@ -27,18 +27,34 @@ public class PubSubMessageReceiverException extends MessageListenerException {
     private final PubSubConsumerState state;
     private final Object listener;
 
+    private boolean autoAcknowledge = true;
```

Comment:
I think [Sonar](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=931&id=micronaut-projects_micronaut-gcp&open=AYqU5co3v55ODOL3QU91) is right in suggesting this field should be `final`.